packages/react-docs/pages/components/tooltip.mdx

```diff
@@ -144,82 +144,104 @@ If the tooltip anchor is not a focusable content, just like the text string, you
+      <Divider orientation="vertical" />
+      <Tooltip
+        hideArrow
+        label={(
+          <Box px="1x" py="2x">
+            <Text>Tooltip Title</Text>
+            <Divider my="2x" />
+            <Text>This is a tooltip</Text>
+          </Box>
+        )}
+        backgroundColor={backgroundColor}
+        color={color}
+      >
+        <Text display="inline-block">Hover me</Text>
+      </Tooltip>
+    </Flex>
   );
 }
 ```
 
+### Placement
+
+Use the `placement` prop to control the placement of the tooltip.
+
+```jsx disabled
+<Tooltip label="Tooltip" placement="top">
+  Tooltip anchor
+</Tooltip>
+```
+
 ```jsx
-function Example() {
-  const textRef = React.useRef();
-  const [isOpen, toggleIsOpen] = useToggle();
-  useEventListener(
-    () => textRef.current,
-    'mouseenter', // It can be either 'mouseenter' or 'pointerenter'
-    React.useCallback((event) => {
-      const isOverflowing = (event.currentTarget.scrollWidth > event.currentTarget.clientWidth);
-      isOverflowing && toggleIsOpen(true);
-    }, [toggleIsOpen]),
-  );
-  useEventListener(
-    () => textRef.current,
-    'mouseleave', // It can be either 'mouseleave' or 'pointerleave'
-    React.useCallback((event) => {
-      toggleIsOpen(false);
-    }, [toggleIsOpen]),
-  );
+<Box display="inline-block">
+  <Grid
+    templateColumns="repeat(3, minmax(80px,1fr))"
+    templateRows="repeat(4, 80px)"
+    gap="8x"
+  >
+    {['top-start', 'top', 'top-end',
+      'right-start', 'right', 'right-end',
+      'bottom-start', 'bottom', 'bottom-end',
+      'left-start', 'left', 'left-end'
+    ].map(placement => (
+      <Tooltip
+        key={placement}
+        label="Tooltip"
+        placement={placement}
+      >
+        <Button width="100%">
+          {placement}
+        </Button>
+      </Tooltip>
+    ))}
+  </Grid>
+</Box>
+```
 
-  return (
-    <Tooltip
-      isOpen={isOpen}
-      label="This is a very long text string that might be truncated when exceeding the container width"
-    >
-      <Flex alignItems="center" columnGap="2x">
-        <Icon icon="menu" />
-        <Truncate ref={textRef}>
-          This is a very long text string that might be truncated when exceeding the container width
-        </Truncate>
-      </Flex>
-    </Tooltip>
-  );
-}
+### Using `PopperProps`
+
+By default, the `OverflowTooltip` component positions the tooltip relative to its parent container. In some cases, the tooltip content might be cut off when it extends outside the container that holds it.
+
+To mitigate this issue, you can pass `PopperProps={{ usePortal: true }}` to `OverflowTooltip` to make it positioned on the document root.
+
+```jsx
+<OverflowTooltip
+  PopperProps={{ usePortal: true }}
+  label="This is a tooltip"
+>
+  This text string will be truncated when exceeding its container width. To see this in action, try resizing your browser viewport. If the text overflows, a tooltip will appear, displaying the full content.
+</OverflowTooltip>
```

Comment:
Should we move this section to `overflowtooltip.mdx` instead?
Since it's focusing on the limitation of `OverflowTooltip`

Reply:
revised in bac1de6